docs/getting_started.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+###############
+Getting started
+###############
+
+Installation
+============
+
+Qiskit Nature depends on the main Qiskit package which has its own
+`Qiskit Getting Started <https://qiskit.org/documentation/getting_started.html>`__ detailing the
+installation options for Qiskit and its supported environments/platforms. You should refer to
+that first. Then the information here can be followed which focuses on the additional installation
+specific to Qiskit Nature .
+
+Qiskit Nature has some functions that have been made optional where the dependent code and/or
+support program(s) are not (or cannot be) installed by default. These include, for example,
+classical libary/programs for molecular problems.
+See :ref:`optional_installs` for more information.
+
+.. tabbed:: Start locally
+
+    The simplest way to get started is to follow the getting started 'Start locally' for Qiskit
+    here `Qiskit Getting Started <https://qiskit.org/documentation/getting_started.html>`__
+
+    In your virtual environment where you installed qiskit simply add nature to the
```

Comment:
```suggestion
    In your virtual environment where you installed Qiskit simply add Qiskit Nature to the
```

Reply:
No that is not what is wanted. where you did 'pip install qiskit` you do `pip install qiskit[nature]` i.e add nature to the list same as visualization example.  I guess its fine either way on qiskit capitalization but maybe I should have quoted nature to make it show up. I'll do that instead